src/Client/V2/IEmsApi.cs

```diff
@@ -849,6 +848,56 @@ public interface IEmsApi
         [Get( "/v2/ems-systems/1/flights/{flightId}/identification" )]
         Task<Identification> GetFlightIdentification( int flightId, [Property] CallContext context = null );
 
+        /// <summary>
+        /// Gets all the airports' information.
+        /// </summary
+        [Get( "/v2/ems-systems/1/navigation/airports" )]
+        Task<NavigationAirport[]> GetNavigationAirports( [Property] CallContext context = null );
+
+        /// <summary>
+        /// Gets the runways for a specific airport.
+        /// </summary>
+        /// <param name="airportId">
+        /// The airport to get the runways for.
+        /// </param>
+        [Get( "/v2/ems-systems/1/navigation/airports/{airportId}/runways" )]
+        Task<NavigationRunway[]> GetNavigationRunways( int airportId, [Property] CallContext context = null );
```

Comment:
Likewise these usually return `IEnumerable` as well so the access class is just a passthrough